apps/nextjs/src/lib/lessonPlan/organiseSections.ts

```diff
@@ -0,0 +1,25 @@
+import { LessonPlanKeys } from "@oakai/aila/src/protocol/schema";
+
+export const organiseSections: {
```

Comment:
I'm still unhappy with this. I don't think it's wise to assume that the LLM will always respond in groups, but this matches the current expected behaviour.